src/components/pages/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,41 @@
+import Pagination from "./Pagination";
+import { mount, shallow } from "enzyme";
+import React from "react";
+import toJson from "enzyme-to-json";
+
+const mockHandlePageClick = jest.fn();
+
+const props = {
+  handlePageClick: mockHandlePageClick,
+  pageCount: 5,
+  currentPage: 2
+};
+
+it("check mounting of pagination component", () => {
+    const component = shallow(<Pagination {...props} />);
+    // @ts-ignore
+    expect(toJson(component)).toMatchSnapshot();
+})
+
+it("check disable of prev button for first page", () => {
+  const component = shallow(<Pagination {...props} currentPage={0}/>);
+  // @ts-ignore
+  expect(component.find(".prev").prop("disabled")).toBe(true)
+})
+
+it("check disable of next button for last page", () => {
```

Comment:
```suggestion
it("should show next button as disabled for last page", () => {
```  Try to make the description of a test case as a complete sentence